src/app/views/datasets-new/edit-metadata-cantabular/CantabularMetadataController.jsx

```diff
@@ -0,0 +1,755 @@
+            dimensionsUpdated: false,
+            datasetMetadataHasChanges: false,
+            versionMetadataHasChanges: false,
+            metadata: {
+                title: "",
+                summary: "",
+                keywords: "",
+                nationalStatistic: false,
+                licence: "",
+                contactName: "",
+                contactEmail: "",
+                contactTelephone: "",
+                relatedDatasets: [],
+                relatedPublications: [],
+                relatedMethodologies: [],
+                releaseFrequency: "",
+                edition: "",
+                version: 0,
+                versionID: "",
+                releaseDate: {
+                    value: "",
+                    error: "",
+                },
+                nextReleaseDate: "",
+                unitOfMeasure: "",
+                notices: [],
+                dimensions: [],
+                usageNotes: [],
+                latestChanges: [],
+                qmi: "",
+            },
+            cantabularMetadata: {
+                dataset: {
+                    qmi: {},
+                    contacts: [
+                        {
+                            name: "",
+                            email: "",
+                            telephone: "",
+                        },
+                    ],
+                },
+                version: {},
+            },
+        };
+    }
+
+    UNSAFE_componentWillMount() {
+        const datasetID = this.props.params.datasetID;
+        const editionID = this.props.params.editionID;
+        const versionID = this.props.params.versionID;
+        this.getMetadata(datasetID, editionID, versionID);
+    }
+
+    getMetadata = (datasetID, editionID, versionID) => {
+        this.setState({ isGettingMetadata: true });
+        return datasets
+            .getEditMetadata(datasetID, editionID, versionID)
+            .then(metadata => {
+                this.setState({ isGettingMetadata: false });
+                this.getCantMetadata(datasetID, metadata);
+            })
+            .catch(error => {
+                this.setState({
+                    isGettingMetadata: false,
+                    disableScreen: true,
+                    allowPreview: false,
+                    disableCancel: false,
+                });
+                log.event("get metadata: error GETting dataset metadata from controller", log.data({ datasetID, editionID, versionID }), log.error());
+                notifications.add({
+                    type: "warning",
+                    message: `An error occured when getting information about this dataset version metadata. Please try refresh the page`,
+                    isDismissable: true,
+                });
+                console.error("get metadata: error GETting dataset metadata from controller", error);
+            });
+    };
+
+    getCantMetadata = (datasetID, nonCantDatasetMetadata) => {
```

Comment:
"Cant" rather unfortunately reads like "Can't" as in "Can not", so it would be better to rename this to `getCantabularMetadata` rather than saving a few key strokes.